src/test/regress/output/multi_size_queries.source

```diff
@@ -0,0 +1,168 @@
+--
+-- SIZE_QUERIES
+--
+-- Test checks whether size of distributed tables can be obtained with citus_table_size.
+-- To find the relation size and total relation size citus_relation_size and 
+-- citus_total_relation_size are also tested.
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1390000;
+ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1390000;
+CREATE TABLE nation_to_size (
+	n_nationkey integer not null,
+	n_name char(25) not null,
+	n_regionkey integer not null,
+	n_comment varchar(152));
+	
+-- Tests on non-distributed tables.	
+SELECT citus_table_size('nation_to_size');
+ERROR:  relation 'nation_to_size' is not distributed
+SELECT citus_relation_size('nation_to_size');
+ERROR:  relation 'nation_to_size' is not distributed
+SELECT citus_total_relation_size('nation_to_size');
+ERROR:  relation 'nation_to_size' is not distributed
+	
+SET citus.replication_model TO DEFAULT;
+SELECT create_distributed_table('nation_to_size', 'n_nationkey');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- Tests when the replication model is not streaming.
+SELECT citus_table_size('nation_to_size');
+ERROR:  Currently, this function only works with distributed tables with replication factor equals to 1 and reference tables.
+SELECT citus_relation_size('nation_to_size');
+ERROR:  Currently, this function only works with distributed tables with replication factor equals to 1 and reference tables.
+SELECT citus_total_relation_size('nation_to_size');
+ERROR:  Currently, this function only works with distributed tables with replication factor equals to 1 and reference tables.
+DROP TABLE nation_to_size;
+SELECT citus_table_size('nation_to_size');
```

Comment:
Let's remove them. Looking into results, it looks like we are getting wrong results.